app/components/courses/apply_component.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="govuk-!-margin-bottom-8" id="section-apply">
+  <% if CycleTimetable.mid_cycle? %>
+    <% if has_vacancies? == 'Yes' %>
```

Comment:
Having a `has_xxx?` method that returns a string is the kind of thing that would trip me up! I'm a bit confused because in the tests it looks like it's stubbed with `true` or `false`.

Reply:
Yes good point @stevehook - we shouldn't be decorating the `#has_vacancies?` 
I've made the change - https://github.com/DFE-Digital/find-teacher-training/pull/1101/commits/cac537401e6eca5dd24fcb5ae3931d323bb2a243